client/modules/food/components/packing/PackingList.js

```diff
@@ -259,6 +260,15 @@ function exists(thing) {
   return thing
 }
 
+function isValid(date) {
```

Comment:
Maybe call this `formatDate` instead. Also the else should be on the same line as the braces - `} else {`

Reply:
Thanks, I’ll make the changes as soon as possible! I’ll try to address the commit issues as wel, not sure what happened there.

Reply:
Just made the suggested changes. Sorry about the weird commits, I'm not sure how they got screwed up but, as you suggested, I'll delete and re-create my fork after this is merged. Hopefully that will give me a blank slate. Thanks!